### ISSUE TITLE
[BUG] Address fields merging not compatible with account address book

### ISSUE THREAD

**Issue by rikwillems** [tig-nl/postcode-magento2#19]
When you split address fields to have separate values for street, housenumber and extension this is not compatible with display in the customer account. 

For this to work nicely we set `customer/address/street_lines` to 1. Otherwise we have 3 street lines in the checkout. The customer account address form now shows 1 line, but without housenumber and extension that were filled in the checkout.

Additionally, going back to the checkout with a saved address, you get an error `"Address" cannot contain more than 1 lines.`

**Comment by tig-jeffreybranderhorst** [comment]
Hi @rikwillems 

First of all thank you for thinking along so we the extension becomes even better.  Making sure the check also works on the account page is standing on our backlog, we will then also pick this up then. 

Thank you again for submitting this.

Have a great day,
Jeffrey

**Comment by Paulsky** [comment]
Is this issue still open? Because I think I have encountered the same problem.

We have set customer/address/street_lines to 2. The extension of the housenumber is saved in address line 3.

This creates a problem when customers who previously checked out with an address with an housenumber extension. They cannot checkout and get an error "Address" cannot contain more than 2 lines."

To fix the problem, the admin has to fix the address. But he can't because he also get's an error '"Address" cannot contain more than 2 lines.'

So the only way to fix this, is to remove the saved address, and manualy add the address again.
